shared/gh/js/controllers/gh.api.util.js

```diff
@@ -139,6 +137,74 @@ define(['exports', 'moment', 'bootstrap-notify'], function(exports, moment) {
     };
 
     /**
+     * Return the current academic week number if the current date is within a term
+     *
+     * @param  {Number}    date     The date where the academic week number needs to be returned for
+     * @return {Number}             The academic week number
+     */
+    var getAcademicWeekNumber = exports.getAcademicWeekNumber = function(date) {
+        if (!_.isNumber(date)) {
+            throw new Error('A valid date should be provided');
+        }
+
+        // Get the configuration
+        var config = require('gh.core').config;
+        // Get the correct terms associated to the current application
+        var terms = config.terms[config.academicYear];
+
+        // Retrieve the corresponding term of the specified date
+        var currentTerm = getTerm(date);
+        if (!currentTerm) {
+            return null;
+        }
+
+        // Get the start date of the corresponding term
+        var startDate = convertISODatetoUnixDate(moment(currentTerm.start).utc().format());
+
+        // Return the current academic week number
+        return Math.ceil((date - startDate) / (PERIODS['week']) - 0.25) + 1;
```

Comment:
Could you provide a bit more documentation around this? Especially around the .25 which doesn't fully make sense tome.